include/frontend/assembler/op-with-info.hpp

```diff
@@ -24,9 +25,12 @@ class OpString {
     std::string_view toString() const {
         return chai::interpreter::OP_TO_STR[val_];
     }
+    chai::interpreter::OperationFormat format() {
+        return chai::interpreter::OP_TO_FORMAT[val_];
+    }
 
     constexpr operator chai::interpreter::Operation() const { return val_; }
-    constexpr auto operator<=>(const OpString &rhs) const = default;
+    constexpr auto operator<=>(const OperationWithInfo &rhs) const = default;
     constexpr auto operator==(const chai::interpreter::Operation &rhs) const {
```

Comment:
@c71n93 Do we need `==` if we have `<=>`?

Reply:
@levBagryansky check types of arguments

Reply:
@c71n93  Looks like extra code: we have already cast from `Operation` to `OperationWithInfo` in line 14, so  `myOperationWtihInfo == myOperation`  will firstly cast `myOperation` to `OperationWtihInfo` and then compare via operator in line 29

Reply:
@levBagryansky done